qiskit_ibm_runtime/qiskit_runtime_service.py

```diff
@@ -85,11 +80,9 @@ def __init__(
             - If a ``filename`` is specified, account details will be loaded from ``filename``,
                 else they will be loaded from the default configuration file.
             - If ``name`` is specified, the corresponding account details will be loaded from
-                the configuration file, including ``channel``, ``token``, ``instance``, ``region``
-                (only ``"ibm_cloud"`` and ``"ibm_quantum_platform"``),
-                ``plans_preference` (only ``"ibm_cloud"`` and ``"ibm_quantum_platform"``),
-                and the advanced configuration parameters: ``url``, ``url_resolver``,
-                ``private_endpoint``,  ``verify``, and  ``proxies``.
+                the configuration file, including ``channel``, ``token``, ``instance``, ``region``,
+                ``plans_preference`, and the advanced configuration parameters: ``url``,
+                ``url_resolver``, ``private_endpoint``,  ``verify``, and  ``proxies``.
```

Comment:
Naive question: do all advanced configuration parameters work with the `ibm_cloud` channels?

Reply:
I don't think so 🥲

Reply:
I think we should keep these parameters for now, I'm not positive they're not used anymore/they don't work with the new API. I'll add a followup